flagsmith/__init__.py

```diff
@@ -1 +1,3 @@
 from .flagsmith import Flagsmith  # noqa
+
+__all__ = ["Flagsmith"]
```

Comment:
1. I believe that, with the explicit export added, the `# noqa` comment can be removed now.

2. Nitpick: I prefer using tuples for the exports as there's no sense in using mutable sequences for those.

Reply:
Removed #noqa, also I tried switching to tuples, but mypy is erroring out and not recognising the exports for some reason.

Reply:
Have you by chance omitted a trailing comma?

```python
# assigns a tuple
__all__ = ("Flagsmith",)

# assigns a string :)
__all__ = ("Flagsmith")
```

Reply:
Ahh, my bad 😣 Yes I missed using a trailing comma! Fixed, thanks!